### PR TITLE
Kraken update

### PR DESCRIPTION
**Pull request by nekrut**
added tests for all Kraken tools using a small custom Kraken db

### CODE REVIEW

File: tool_collections/kraken/kraken/kraken.xml [hunk 1]
```diff
@@ -47,7 +50,10 @@
         #if $split_reads:
             --classified-out "${classified_out}" --unclassified-out "${unclassified_out}"
         #end if
-        --output "${output}"
+
+        ## The --output option was changed to redirect as it does not work properly is some situations
```
Comment (bgruening) [inline tool_collections/kraken/kraken/kraken.xml]: Anton can we please increase the version number of this tool. I guess this is needed if change the output ...

**Comment by bgruening** [comment]
@nekrut are the test databases the same for all tools or are they distinct. Just asking if we can save space if we symlink them.

**Comment by nekrut** [comment]
@bgruening the database is exactly the same. We can symlink them. I'll increment the version number.

**Comment by bgruening** [comment]
Great! Symlinking is great if we can not use demultiplexing. Please rebase this PR so we do not pollute the github repo.

**Comment by nekrut** [comment]
I'll wait for @martenson and rebase

**Comment by nekrut** [comment]
Rebased with @martenson 's help

**Comment by davebx** [comment]
:+1:

File: tool_collections/kraken/kraken/kraken.xml [hunk 2]
```diff
@@ -47,7 +50,10 @@
         #if $split_reads:
             --classified-out "${classified_out}" --unclassified-out "${unclassified_out}"
         #end if
-        --output "${output}"
+
+        ## The --output option was changed to redirect as it does not work properly is some situations. For example, on test database the tool classifies 4 reads but does not write them into a file if --output is specified. It does however print correct output into STDOUT. This behavior can be re-created with test database provided in test-data/test_db/ folder. This is the reason for incrementing version number from 1.1.2 ro 1.1.3
```
Comment (blankenberg) [inline tool_collections/kraken/kraken/kraken.xml]: minor: 'ro' to 'to'

Comment (bgruening) [inline tool_collections/kraken/kraken/kraken.xml]: I will change this post-merge.

**Comment by bgruening** [comment]
Thanks @nekrut and all reviewers!

**Pull request merged**